src/snapred/backend/data/LocalDataService.py

```diff
@@ -490,12 +486,19 @@ def readNormalizationRecord(self, runId: str, version: str = None):
             latestFile = self._getLatestFile(recordPath)
         record: NormalizationRecord = None  # noqa: F821
         if latestFile:
+            print(f"reading NormalizationRecord from {latestFile}")
```

Comment:
log?

Reply:
Actually, that wasn't one of ours.  :)   I'll take a look and make sure there aren't any more of them in `LocalDataService`.

Reply:
I ended up moving most of these to `logger.info(...)`.  I still am confused about why they were their in the first place.  :(